### DISCUSSION TITLE
Quando veremos o serviço https://dify.ai/

### DISCUSSION THREAD

**Discussion by jeffcoludi** [coollabsio/coolify#3926] [Service Template Requests]
Quero fazer lembrar a Equipe geniosa do Coolify sobre a possibilidade de integrar o serviço Dify Ai, com certeza teremos muitas expectativas nessa nova integração. https://docs.dify.ai/getting-started/install-self-hosted/docker-compose

**Comment by jeffcoludi** [comment]
https://docs.dify.ai/getting-started/install-self-hosted/docker-compose

**Comment by sacanods** [comment]
I have the same needs.
